test/functional/core/job_spec.lua

```diff
@@ -138,7 +138,7 @@ describe('jobs', function()
   end)
 
   it('preserves NULs', function()
-    if helpers.pending_win32(pending) then return end  -- TODO: Need `cat`.
+    if helpers.pending_win32(pending) then return end   -- XXX: Hangs in Appveyor
```

Comment:
in https://github.com/neovim/neovim/pull/6129 i left this note:

```lua
-- TODO: Windows: why is the data received on_stderr instead of on_stdout?
if helpers.pending_win32(pending) then return end
```

Reply:
There is `set shellredir=>` in `set_shell_powershell`.

Reply:
Aha! Nice catch.